buildSrc/src/main/java/com/google/firebase/gradle/plugins/PublishingPlugin.kt

```diff
@@ -373,7 +373,10 @@ abstract class PublishingPlugin : Plugin<Project> {
       printReleaseConfig.convention(project.provideProperty("printOutput"))
 
       releaseConfigFile.convention(project.layout.projectDirectory.file(RELEASE_CONFIG_FILE))
-      releaseReportFile.convention(project.layout.projectDirectory.file(RELEASE_REPORT_FILE))
+      releaseReportMdFile.convention(project.layout.projectDirectory.file(RELEASE_REPORT_MD_FILE))
```

Comment:
Is the goal to eventually get rid of the `.md` file, in favor of the parsable `.json` (and possibly the dashboard)?

Reply:
correct